.circleci/config.yml

```diff
@@ -437,35 +450,54 @@ jobs:
             --form "file=@${fileName}" \
 
   instrumentation_tests_device:
-    macos:
-      xcode: "12.3.0"
-    resource_class: large
+    resource_class: xlarge
```

Comment:
Instrumentation tests are now faster than the overall longest job-chain (unit tests + sonar), so we could save some resources by preventing firebase from spinning up if trivial checks fails, i.e.

```yml
      - instrumentation_tests_device:
          requires:
            - ktlint_device_release_check
            - detekt
```

Thoughts?

Reply:
Good point

Reply:
Good point